src/super_gradients/training/utils/segmentation_utils.py

```diff
@@ -119,6 +120,11 @@ def visualize_batch(
 
         :param undo_preprocessing_func: a function to convert preprocessed images tensor into a batch of cv2-like images
         :param image_scale:             scale factor for output image
+
+        Parameters
+        ----------
+        pred_mask
```

Comment:
We use `param PARAM NAME: PARAM DESCRIPTION` docstrings format. Please update the docstrings, otherwise the documentation on the website may break

Reply:
This is not my change and I wasn't working on this function. I can change it if you would like me to do it.

Reply:
@Yael-Baron there is a mismatch in docstring indeed, `pred_boxes` and `target_boxes` are never used.
It would be great if you can remove the new ```Parameters
        ----------``` and replace `pred_boxes -> pred_mask` , `target_boxes -> target_mask ` 🙏

Reply:
Fixed on 6e8a1be55dc60d7f9c3bb9fe618d0e4bb5127df4